doc_src/cmds/exit.rst

```diff
@@ -13,6 +13,6 @@ Synopsis
 Description
 -----------
 
-``exit`` causes fish to exit. If ``STATUS`` is supplied, it will be converted to an integer and used as the exit status. Otherwise, the exit status will be that of the last command executed.
+``exit`` causes fish to exit. If ``STATUS`` is supplied, it will be converted to an integer, clamped at 255, and used as the exit status. Otherwise, the exit status will be that of the last command executed.
```

Comment:
I wonder if it's even worth to document this because users should not need to know.
Encoding more than a few values in a return code is likely the wrong approach, so if someone relied on this, they should probably use something else.

Reply:
That's fair; "exit will return valid exit codes" is probably overkill. Reverted, along with the documentation for `return`.